align_system/cli/run_align_system.py

```diff
@@ -272,7 +272,7 @@ def _compute_time_stats(times_s):
                 # prevent ADMs from modifying the originals (should
                 # considering doing the same for current_state and
                 # alignment_target)
-                action_to_take = adm.choose_action(
+                action_to_take, choice_info = adm.choose_action(
```

Comment:
Would prefer to have this section of code work with either a single value or tuple return (to be more backward compatible and foolproof); would also need to update the type annotations here: https://github.com/ITM-Kitware/align-system/blob/main/align_system/algorithms/abstracts.py#L13

Reply:
(FWIW even without supporting the different return shapes here, would want to update the type annotations)

Reply:
Makes sense, I added the backward support here https://github.com/ITM-Kitware/align-system/pull/122/commits/44086be555f3110f439bcf106fdfd1705f37437b and updated the abstract type annotation here https://github.com/ITM-Kitware/align-system/pull/122/commits/6684d9e825aa15c1703500165d147d898ac9e37e

Reply:
The updated code LGTM, thanks for making this change